governance/votes/Executive vote - May 25, 2022.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Template - [Executive Vote] Raise the ESM Minimum Threshold, MKR Vesting, Adjust the lid Parameter on the Rate-limited Flapper - May 25, 2022
+summary: Raising the Emergency Shutdown Module Minimum Threshold. MKR Vesting Distribution to DUX-001 and SAS-001. Adjusting the `lid` parameter from 150,000 MKR to 30,000 MKR.
+date: 2022-05-25T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Raise the ESM Minimum Threshold, MKR Vesting, Adjust the lid Parameter on the Rate-limited Flapper - May 25, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The [Emergency Shutdown Module (ESM)](https://makerdao.world/en/learn/governance/emergency-shutdown) Minimum Threshold (`min`) will be increased from 100,000 MKR to **150,000 MKR**.
+- **599,6553 MKR** will be distributed to multiple Core Units, as detailed further down.
+- The `lid` parameter will be adjusted on the [Rate-limited Flapper](https://forum.makerdao.com/t/rate-limited-flapper/13056), as detailed further down.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### ESM Minimum Threshold Increase
+
+As per [this Governance Poll](https://vote.makerdao.com/polling/QmQgab4a), the ESM Minimum Threshold (`min`) ---i.e., the minimum MKR amount required for triggering an [Emergency Shutdown](https://makerdao.world/en/learn/governance/emergency-shutdown)--- will be increased as follows if this executive proposal passes:
+
+- From 100,000 MKR to **150,000 MKR**.
```

Comment:
Can you do a "For further information please see the forum thread" and link to the Signal Request?